fix: Fixes inconsistent result when using a multi-region cluster by always using a single spec (#2685)
mongodb/terraform-provider-mongodbatlas: .changelog/2685.txt

```release-note:bug

> resource/mongodbatlas_search_deployment: Fixes inconsistent result when using a multi-region cluster by always using a single spec

Reviewer: ```suggestion
resource/mongodbatlas_search_deployment: Fixes inconsistent result for a multi-region cluster that always uses a single spec.
```
